test/index.test.js

```diff
@@ -1,38 +1,124 @@
 import { h, app } from "hyperapp"
-import { Route, location } from "../src"
+import { Route, Link, Redirect, location } from "../src"
 
-test("Router", done => {
-  const state = {
-    location: location.state
-  }
+const wait = async ms => new Promise(resolve => setTimeout(resolve, ms))
+const click = e => e.dispatchEvent(new MouseEvent("click", { button: 0 }))
+
+let state, actions, unsubscribe
```

Comment:
I try to avoid using `var` and `let` when writing pure ES6, but I can fix this later!